version/version.go

```diff
@@ -1,51 +1,68 @@
 package version
 
 import (
-	"bufio"
+	"encoding/json"
 	"fmt"
 	"net/http"
 	"strings"
 
 	"github.com/blang/semver"
+	"github.com/hashicorp/go-cleanhttp"
 	"github.com/urfave/cli"
 )
 
-// Version represents the value of the current semantic version
+// Version represents the value of the current semantic version.
 var Version = "3.7.3"
 
-// PrintVersion handles the version command for sops
+// PrintVersion prints the current version of sops. If the flag
+// `--disable-version-check` is set, the function will not attempt
+// to retrieve the latest version from the GitHub API.
+//
+// If the flag is not set, the function will attempt to retrieve
+// the latest version from the GitHub API and compare it to the
+// current version. If the latest version is newer, the function
+// will print a message to stdout.
 func PrintVersion(c *cli.Context) {
-	out := fmt.Sprintf("%s %s", c.App.Name, c.App.Version)
+	out := strings.Builder{}
+
+	out.WriteString(fmt.Sprintf("%s %s", c.App.Name, c.App.Version))
+
 	if c.Bool("disable-version-check") {
-		out += "\n"
+		out.WriteString("\n")
 	} else {
-		upstreamVersion, err := RetrieveLatestVersionFromUpstream()
-		if err != nil {
-			out += fmt.Sprintf("\n[warning] failed to retrieve latest version from upstream: %v\n", err)
-		}
-		outdated, err := AIsNewerThanB(upstreamVersion, Version)
+		upstreamVersion, upstreamURL, err := RetrieveLatestReleaseVersion()
 		if err != nil {
-			out += fmt.Sprintf("\n[warning] failed to compare current version with latest: %v\n", err)
+			out.WriteString(fmt.Sprintf("\n[warning] failed to retrieve latest version from upstream: %v\n", err))
 		} else {
-			if outdated {
-				out += fmt.Sprintf("\n[info] sops %s is available, update with `go get -u github.com/getsops/sops/v3/cmd/sops`\n", upstreamVersion)
+			outdated, err := AIsNewerThanB(upstreamVersion, Version)
+			if err != nil {
+				out.WriteString(fmt.Sprintf("\n[warning] failed to compare current version with latest: %v\n", err))
 			} else {
-				out += " (latest)\n"
+				if outdated {
+					out.WriteString(fmt.Sprintf("\n[info] a new version of sops (%s) is available, you can update by visiting: %s\n", upstreamVersion, upstreamURL))
+				} else {
+					out.WriteString(" (latest)\n")
+				}
 			}
 		}
 	}
-	fmt.Fprintf(c.App.Writer, "%s", out)
+	fmt.Fprintf(c.App.Writer, out.String())
```

Comment:
I think
```suggestion
	fmt.Fprintf(c.App.Writer, "%s", out)
```
is better since it won't process `%` in `out.String()`, if these appear for some reason. These could have been part of `err` for example.